src/app/shared/components/template/components/button/button.component.html

```diff
@@ -16,3 +19,24 @@
     ></plh-template-component>
   </span>
 </ion-button>
+
+<div
+  class="button-container"
```

Comment:
nit(non-blocking) probably good to include the variant name in the class to make css more explicit and then nest all child styles within the `.variant-button {....} ` top style